ruby-challenges.rb

```diff
@@ -19,17 +29,47 @@
 nums_array2 = [25, 17, 47, 11]
 # Expected output: 100
 
+def returns_sum (array_of_nums)
+    array_of_nums.inject(:+)
+end
+p returns_sum(nums_array1)
+p returns_sum(nums_array2)
```

Comment:
Awesome!  Ruby has another built-in method that is for newer Ruby versions called .sum() that has the same effect.